packages/core/src/postProcess/PostProcessPass.ts

```
@@ -0,0 +1,83 @@
+   * @param camera - The camera used to render
+   * @param srcTexture - The source texture from last render target
+   * @param destTarget - The destination render target
+   */
+  abstract onRender(camera: Camera, srcTexture: Texture2D, destTarget: RenderTarget): void;
+
+  /**
+   * @inheritdoc
+   */
+  override _onDestroy() {
+    super._onDestroy();
+    this._postProcessManager && this._postProcessManager._removePostProcessPass(this);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)